src/pages/Withdrawals/index.tsx

```diff
@@ -0,0 +1,1079 @@
+            <SectionTitle level={2} className="mb10">
+              <FormattedMessage defaultMessage="Introduction" />
+            </SectionTitle>
+          </Anchor>
+          <Text className="mb10">
+            <FormattedMessage
+              defaultMessage="The Shanghai/Capella upgrade enabled Beacon Chain staking
+              withdrawals, with the inclusion of EIP-4895. This Ethereum improvement proposal
+              provides a mechanism of continuously distributing rewards to stakers, and also
+              providing a way for exited validators to unlock their entire balance—no gas
+              required."
+            />
+          </Text>
+
+          <section>
+            <Anchor to="#enabling-withdrawals" id="enabling-withdrawals">
+              <SectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="Enabling withdrawals" />
+              </SectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="For your validator account to be capable of withdrawals of any
+                kind, a withdrawal address must be provided and registered to your account on the
+                Beacon Chain. This should be an address you control, and cannot be changed once set."
+              />
+            </Text>
+
+            <WithdrawalsTabComparison />
+          </section>
+
+          <section>
+            <Anchor to="#update-your-keys" id="update-your-keys">
+              <SectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="How to update validator keys" />
+              </SectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="If your validator account still has BLS withdrawal credentials ({type0}),
+                a one-time message must be broadcast signaling which execution layer account should be
+                used for all withdrawals."
+                values={{ type0: <Code>0x00</Code> }}
+              />
+            </Text>
+
+            <section className="actionable">
+              <Heading level={4} className="mb10">
+                <FormattedMessage defaultMessage="Tools available" />
+              </Heading>
+              <ul>
+                <li>
+                  <strong>ethdo</strong> -{' '}
+                  <Link
+                    inline
+                    primary
+                    to="https://github.com/wealdtech/ethdo/blob/master/docs/changingwithdrawalcredentials.md"
+                  >
+                    Changing withdrawal credentials
+                  </Link>
+                </li>
+                <li>
+                  <strong>Staking Deposit CLI</strong> -{' '}
+                  <Link
+                    inline
+                    primary
+                    to="https://notes.ethereum.org/@hww/btec"
```

Comment:
Moved to https://notes.ethereum.org/@launchpad/btec
```suggestion
                    to="https://notes.ethereum.org/@launchpad/btec"
```